build.sbt

```diff
@@ -9,6 +9,7 @@ import local.Scripted
 def buildLevelSettings: Seq[Setting[_]] =
   inThisBuild(
     Seq(
+      doc in Compile := file("/dev/null"),
```

Comment:
What's the purpose of this?

Reply:
https://github.com/sbt/sbt/pull/4335/commits/ba6d0fb4a7d799f6c1011be199a36370414090ac should be reverted. I was using this to speed up local development but didn't mean to include it in the PR.